### ISSUE TITLE
Recipes broken?

### ISSUE THREAD

**Issue by DjebbZ** [roman01la/uix#9]
In `dev/uix/recipes.cljs` the command `clojure -A:dev -m uix.recipes.server-rendering` produces a an HTML broken. It consists just of the "Lorem ipsum" text repeated all over the page. I see no select to choose a recipe.

**Comment by DjebbZ** [comment]
To be more precise both `/` and `/chunked` display this pattern of text:

``` html
4Lorem ipsum...
<!-- pattern start (note: this is my comment, not part of the HTML) -->
<div>3Lorem ipsum
  <div>2Lorem ipsum
    <div>1Lorem ipsum</div>
    <div>1Lorem ipsum</div></div>
  <div>2Lorem ipsum
    <div>1Lorem ipsum</div>
    <div>1Lorem ipsum</div></div></div>
<!-- pattern end -->
```

The pattern seems to repeat a lot of time, and the text starting with `4` appears only once at the very beginning. Seems like a recursion problem.

**Comment by roman01la** [comment]
SSR recipe doesn't have a front-end part, it's just an example of server side code. It recursively generates lots of HTML and sends it to the client.

**Comment by DjebbZ** [comment]
Yes, I've just understood this. My bad. Is there a way to see the front-end recipe in a browser?

**Comment by roman01la** [comment]
Do you mean something that renders React on top of SSRed markup? No, right now there's nothing like this in recipes. I might put it later, if that's important.

**Comment by DjebbZ** [comment]
Not necessarily an isomorphic example (even if I think it's important). Just a way to run the `uix.recipes` in `dev/`.

**Comment by DjebbZ** [comment]
Ok I found it: `clojure -A:dev -m figwheel.main --build dev --repl --serve`. Should I send you a PR to add it to the README ?

**Comment by DjebbZ** [comment]
I could instead send a PR to add an alias that runs this directly (same for the server-side rendering recipe) ?

**Comment by DjebbZ** [comment]
It's done then :)

**Comment by roman01la** [comment]
I've ported front-end recipes to SSR as well.
